.travis.yml

```
@@ -1,12 +1,7 @@
 language: rust
```

```suggestion
language: rust
rust:
    - nightly
    - nightly-2019-01-02
script: |
  cargo build --verbose &&
  cargo test  --verbose --all
matrix: 
  include:
  - rust: nightly-2019-02-02
    env: CLIPPY=YESPLEASE
    before_script: |
      rustup component add clippy-preview
      rustup component add rustfmt-preview
    script: |
      cargo clippy --all -- -D warnings
      cargo fmt --all -- --check
```

I suggest something like this which pins on a current nightly for `rustfmt` and `clippy`

---

@spacekookie Does this build the project with nightly too? The README for the project seemed to indicate a desire to stay on stable Rust. I'd like to build and test on stable, but use clippy and rustfmt on nightly.

---

I tried to build it on `stable` yesterday and it failed so I assumed that it would always require nightly?
If that was wrong, then yea adding `stable` to the rust list is gonna be good!

---

This happens for me

```rust
  ❤ (azedes) ~/P/c/void> cargo build
   Compiling voidmap v1.1.5 (/home/spacekookie/Personal/clones/void)
error[E0554]: #![feature] may not be used on the stable release channel
  --> src/lib.rs:3:1
   |
3  | / #![feature(
4  | |     box_patterns,
5  | |     infer_static_outlives_requirements,
6  | |     nll,
...  |
10 | |     type_ascription
11 | | )]
   | |__^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0554`.
error: Could not compile `voidmap`.

To learn more, run the command again with --verbose.
```

---

Wow, I'm dumb. I was looking at my other PR.

Actually, it's probably better to look at that one since it folds in most of these changes anyway.

---

Which PR?

---

#24

---

Ah right, I've started reviewing that one then. Do you wanna close this PR then?
The other one seems to be in much better shape.

Maybe just add the `.travis.yml` changes I suggested here